addons/website_event_track/static/src/js/event_track_timer.js

```diff
@@ -0,0 +1,124 @@
+odoo.define('website_event_track.event_track_timer', function (require) {
```

Comment:
Wow..
All this countdown thing seems way over-engineerd.

Can't we simply use setInterval to countdown?
There is an example of a timer we already made in "survey_timer.js.

Reply:
Using a singleton might be an anti-pattern in Odoo. So, I made a rework: The timer will now be a new widget that can be attached to another widget. The timer will basically emit a new event every second and the other widget will have the responsibility to update the dom when an event is triggered.